### DISCUSSION TITLE
Forge test -m doesn't work

### DISCUSSION THREAD

**Discussion by cursedId0l** [Cyfrin/foundry-full-course-cu#162] [Q&A] [answered]
When I try to use `forge test -m` it never works for me. I see patrick doing it all the time in the course
     
     
![image](https://github.com/Cyfrin/foundry-full-course-f23/assets/85530348/207041e0-f711-4b52-ae62-dd4402b0afaa)

Also I don't see it in the help section

![image](https://github.com/Cyfrin/foundry-full-course-f23/assets/85530348/85b0916d-77b9-4389-a90f-1ab4c2f30cc1)

Maybe I am running an older version version of foundry but I thought I installed it fresh when I started the course.

![image](https://github.com/Cyfrin/foundry-full-course-f23/assets/85530348/fbda750e-35db-40a0-9422-311275c5c631)

**Comment by ghost** [comment]
Hello @jtardioli refer to discussion [112](https://github.com/Cyfrin/foundry-full-course-f23/discussions/112) to see the solution.

**Comment by cromewar** [comment, marked as answer]
Please @jtardioli refer to the chronological issues [here](https://github.com/Cyfrin/foundry-full-course-f23/blob/main/chronological-updates.md#lesson-7).
